br/pkg/restore/misc.go

```diff
@@ -64,23 +69,56 @@ func GetTableSchema(
 	return table.Meta(), nil
 }
 
-// GetExistedUserDBs get dbs created or modified by users
-func GetExistedUserDBs(dom *domain.Domain) []*model.DBInfo {
+const MAX_USERTABLES_NUM = 10
```

Comment:
this is not golang's style

maxUserTablesNum

Reply:
done

Reply:
done